recipes/easymt/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "easymt" %}
+{% set version = "1.0.0" %}
+package:
+  name: {{name}}
+  version: {{version}}
+
+source:
+  url: https://github.com/tang-shuyuan/EeayMT/releases/download/v{{version}}/{{name}}_v{{version}}.tar.gz
+  sha256: d9220c8399eaba7afeae9de7853f3c7a59db47c19669a9101f4b5016750b3d98
+
+build: 
+  number: 0
+  entry_points: 
+    - easymt = extract:main
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('easymt', max_pin="x") }}
+
+requirements: 
+  bulid: 
+    - python >=3.9
+  run: 
+    - blast =2.15.0
+    - flye >=2.9.4
+test: 
+  source_files: 
+    - extract.py
+    - version.py
+    - Arabidopsis_protein.fasta
+    - Drosophila_gunungcola_CM045947.fasta
+  commands:
+    - easymt -h
+
+about: 
+  home: https://github.com/tang-shuyuan/EeayMT
```

Comment:
In this github repo is only a tar.gz. This is not how we usually operate. Can you please check in your code into this repo and then do a proper release?

Reply:
actually, this is a newly developed pipeline. Our code is very concise.However, our strategy is highly effective. The script has already been uploaded to my repository.